qubes/dz.Convert

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+python -m dangerzone.conversion.doc_to_pixels_qubes_wrapper
```

Comment:
Add here a new line.

Reply:
Done in commit https://github.com/freedomofpress/dangerzone/pull/437/commits/3c595ea20da95dadac10f2d308131fb70397669d